src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -67,6 +69,11 @@ default String getAsTextShowAll() {
             builder.append(detailIsPrivate);
         }
         builder.append(getAddress())
+                .append(" Age: ");
+        if (getAge().isPrivate()) {
```

Comment:
Not sure what you are going for here. Age is retrieved regardless of whether it is private. Remember to use guard clause effectively.